cmd/dag.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/ghodss/yaml"
 	"github.com/qri-io/dag"
 	"github.com/qri-io/ioes"
+	"github.com/qri-io/qri/fsi/component"
```

Comment:
weird to see `fsi` in this part of the codebase. We should think about moving `component` into `base` in the near future

Reply:
Agreed, will move that soon, it's not strictly a part of fsi.